plugins/SitesManager/vue/src/SitesManagement/SitesManagement.vue

```diff
@@ -67,21 +67,29 @@
       <div class="ui-confirm add-site-dialog">
         <div>
           <h2>{{ translate('SitesManager_ChooseMeasurableTypeHeadline') }}</h2>
-
           <div class="center">
-            <p>
+            <p>{{ translate('SitesManager_ChooseMeasurableTypeSubheader') }}</p>
+            <br>
```

Comment:
Since we already target `.card-row` in the styling, it might be cleaner to give it some padding top instead of using a line break here to create more space.